en/sensor/px4flow.md

```diff
@@ -82,7 +100,36 @@ Address | Bit 2 | Bit 1 | Bit 0
  :  |  :  |  :  |  :
 7 Bit Address 0x49 |  1  |  1  |  1
 
-> **Tip** There are different I2C readouts available. Details about the **I2C frame** and the **I2C integral frame** are in the driver source code.
+> **Tip** There are different I2C readouts available. 
+  Details about the **I2C frame** and the **I2C integral frame** are in the driver source code.
+
+
+If you modify the I2C address of the PX4Flow, make sure to start the PX4 driver with the correct address:
+```
+px4flow start                  # address=0x42 (default)
+px4flow stop
+px4flow start -a 0x45          # address=0x45
+```
+
+
+## Focusing the Lens
+
```

Comment:
Was not sure this is strictly necessary - it was in the devguide. However below somewhere we mention that the optics arrive focussed. I suspect this is useful though?